playbooks/tuf-on-ci/SIGNER.md

```diff
@@ -1,6 +1,22 @@
-# Signer manual for Sigstore root-signing
+# Keyholder manual for Sigstore root-signing
 
-## One-time setup for new signers
+## Keyholder requirements
+
+1. Keyholders MUST subscribe to the [Sigstore Maintainer Calendar](https://calendar.google.com/calendar/u/0?cid=Y19ydjIxcDJuMzJsbmJoYW5uaXFwOXIzNTJtb0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) for upcoming root signing events.
```

Comment:
Is this calendar still accurate? Do we want to deprecate it entirely and just rely on GitHub notifications from TUF-on-CI?

Reply:
I just copy-pasted, happy to remove if no-one maintains the calendar: The generated PRs and the repository status page could be a somewhat decent replacement.

Reply:
asked for opinions on keyholders slack, will remove if no objections

Reply:
removing